README.md

```diff
@@ -1,4 +1,4 @@
-# pullre-kun
+# pullre-kun (Forked)
```

Comment:
Can you remove "(Forked)"?

Reply:
Ok, I reverted Readme.md and pushed the modification.  

Sorry for adding that. I was testing pull requests with the app. but I should've done it with different repo.